packages/start/src/client/createServerFn.ts

```diff
@@ -17,8 +17,6 @@ import type {
   ResolveAllValidators,
 } from './createMiddleware'
 
-//
```

Comment:
i thought these were a typo added by me but apparently they were there before - happy to add them back if they're there for a reason